tests/components/sonarr/__init__.py

```diff
@@ -5,6 +5,6 @@
 MOCK_REAUTH_INPUT = {CONF_API_KEY: "test-api-key-reauth"}
 
 MOCK_USER_INPUT = {
-    CONF_URL: "http://192.168.1.189:8989",
+    CONF_URL: "http://192.168.1.189:8989/",
```

Comment:
why the extra slash?

Reply:
My assumption is that at least a few users will probably have this proxied behind some kind of service an may have to have a path component, if no path is specified then yarl just does a `/` in place of it.

reference:  https://github.com/home-assistant/core/pull/127625/files#diff-e4368ac4899b15f980f4e784edaa60285fd6f7b04847463d414d86e17f4a2e87R105
```
>>> url = yarl.URL("https://my.exmaple.com")
>>> url.path
'/'
```